pkg/egressgateway/manager.go

```diff
@@ -32,6 +34,13 @@ var (
 	zeroIPv4 = net.ParseIP("0.0.0.0")
 )
 
+var Cell = cell.Module(
```

Comment:
Since this is what code navigation will jump you to as an entry point for this file, would be nice to have a comment here explaining what this cell provides